distributed/deploy/ssh.py

```diff
@@ -295,7 +295,8 @@ def __init__(self, scheduler_addr, scheduler_port, worker_addrs, nthreads=0, npr
         self.memory_limit = memory_limit
         self.worker_port = worker_port
         self.nanny_port = nanny_port
-        self.remote_dask_worker = remote_dask_worker
+        if remote_dask_worker is None:
+            self.remote_dask_worker = 'distributed.cli.dask_worker'
```

Comment:
When is it `None`?  Why wouldn't the default value be used?

Also, this implementation is a little bit buggy, you don't do anything with the value if it is not None.

Reply:
None is the default coming from the cli script.

Reply:
But perhaps this should be fixed in <https://github.com/dask/distributed/blob/master/distributed/cli/dask_ssh.py#L50>?

Reply:
Yes, I think I'd prefer that we handle this there if possible.  In general I'd prefer to avoid hard-coding the default values for keywords twice.

Reply:
I've reverted these and rewritten `cli.dask_ssh` to handle this using kwargs.